spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validation' do
+    context 'presence' do
+      it { should validate_presence_of(:username) }
```

Comment:
Change to use the expect syntax for consistent with others
```suggestion
      it { is_expected.to validate_presence_of(:username) }
```

Reply:
Fixed on 1494bae628ebd3c232d9ce29003ac802ed164223